api/classes/base_controller.js

```diff
@@ -36,6 +36,10 @@ BaseController.prototype.create = function(req, reply) {
       if (!record) { throw Boom.notFound(); }
       return req.generateResponse(record.toJSON())
         .code(201);
+    })
+    .catch(function(e) {
+      // We might want to parse the error types here.
+      reply(e);
```

Comment:
So far, this will do *exactly* what happened before, with the benefit that we can debug/log here to see what the error object is.